octree_web_viewer/client/octree_viewer.ts

```diff
@@ -274,13 +277,19 @@ export class OctreeViewer {
 
     this.nodeLoader = new NodeLoader();
     this.currentlyLoading = 0;
+    this.octree_id = 'init_id';
+  }
+
+  public load_new_tree(octree_id: string) {
```

Comment:
I think we shall leave setting the octree_id only at constructor part and not exposing a public function for setting it.

Reply:
done!